open-metadata-implementation/user-interfaces/ui-admin-services/ui-admin-services-server/src/main/java/org/odpi/openmetadata/userinterfaces/adminservices/UIServerAdminStoreServices.java

```diff
@@ -0,0 +1,316 @@
+/* SPDX-License-Identifier: Apache 2.0 */
+/* Copyright Contributors to the ODPi Egeria project. */
+package org.odpi.openmetadata.userinterfaces.adminservices;
+
+
+import org.odpi.openmetadata.adapters.repositoryservices.ConnectorConfigurationFactory;
+import org.odpi.openmetadata.commonservices.ffdc.rest.VoidResponse;
+import org.odpi.openmetadata.frameworks.connectors.Connector;
+import org.odpi.openmetadata.frameworks.connectors.ConnectorBroker;
+import org.odpi.openmetadata.frameworks.connectors.properties.beans.Connection;
+import org.odpi.openmetadata.userinterface.adminservices.configuration.properties.UIServerConfig;
+import org.odpi.openmetadata.userinterface.adminservices.ffdc.UIAdminErrorCode;
+import org.odpi.openmetadata.userinterface.adminservices.ffdc.exception.UIInvalidParameterException;
+import org.odpi.openmetadata.userinterface.adminservices.ffdc.exception.UINotAuthorizedException;
+import org.odpi.openmetadata.userinterface.adminservices.rest.ConnectionResponse;
+import org.odpi.openmetadata.userinterface.adminservices.store.UIServerConfigStore;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * UIServerAdminStoreServices provides the capability to store and retrieve configuration documents.
+ *
+ * A configuration document provides the configuration information for a server.  By default, a
+ * server's configuration document is stored in its own file.  However, it is possible to override
+ * the default location using setConfigurationStoreConnection.  This override affects all
+ * server instances in this process.
+ */
+public class UIServerAdminStoreServices
+{
+    private static Connection  configurationStoreConnection = null;
+
+    private static final Logger log = LoggerFactory.getLogger(UIServerAdminStoreServices.class);
+
+    private UIServerExceptionHandler   exceptionHandler = new UIServerExceptionHandler();
+    private UIServerErrorHandler errorHandler = new UIServerErrorHandler();
+
+    /**
+     * Override the default location of the configuration documents.
+     *
+     * @param userId calling user.
+     * @param connection connection used to create and configure the connector that interacts with
+     *                   the real store.
+     * @return void response
+     */
+    public synchronized VoidResponse setConfigurationStoreConnection(String       userId,
+                                                                     Connection   connection)
+    {
+        final String methodName = "setConfigurationStoreConnection";
+
+        log.debug("Calling method: " + methodName);
+
+        VoidResponse response = new VoidResponse();
+
+        try
+        {
+            //OpenMetadataPlatformSecurityVerifier.validateUserAsOperatorForPlatform(userId);
```

Comment:
Why commented out - this is reason why we need to minimize the differences between the platforms otherwise inconsistencies like this creep in.